backend/src/main/java/gov/cdc/usds/simplereport/config/FeatureFlagsConfig.java

```diff
@@ -0,0 +1,43 @@
+package gov.cdc.usds.simplereport.config;
+
+import gov.cdc.usds.simplereport.db.model.FeatureFlag;
+import gov.cdc.usds.simplereport.db.repository.FeatureFlagRepository;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "features")
+@EnableScheduling
+@Setter
+@Getter
```

Comment:
I believe that instead of setting these at the class level, you could just set them for the `multiplexEnabled` field. If I'm understanding the `AccessLevel.NONE` enum correctly, that would accomplish the same thing - no getters/setters on the _repo, but you'd still have them for the `multiplexEnabled` bool.

Reply:
My logic behind adding it at the class level is that when the config object grows we will not need to add the getter and the setter annotation at the field level. I am predicting that we will need to be able to access the individual flags /fields across the app (when using flags in the Java code) and the repo object is the only one that should stay private in the class. 

So that is why I went with the approach of add all fields minus _repo.

Reply:
I see, that makes sense for now. Thank you!